### ISSUE TITLE
NuGet package is not optimized

### ISSUE THREAD

**Issue by abogarsukov** [Real-Serious-Games/C-Sharp-Promise#84]
For some reason [NuGet package](https://www.nuget.org/packages/RSG.Promise/3.0.1) for the project contains non-optimized (Debug) assembly RSG.Promise.dll Is this intended? If not, please upload a new version with the release assembly.

**Comment by Arvtesh** [comment]
Any updates on this one? It seems logical to have release assemblies in NuGet package.

**Comment by RoryDungan** [comment]
Yeah this should probably be changed to the release one. There wasn't any particular reason I set it up that way apart from that it was the default config and I do sometimes find it useful being able to just copy the PDB files into a project and debug C-Sharp-Promise from there. Obviously that's a bit of a niche use-case because it only applies to people working on C-Sharp-Promise and for most people having a smaller DLL is better.

I don't work at RSG any more but if someone wanted to make a PR that changes the Nuget config to use release assemblies I don't see any reason why we couldn't merge it and push a new minor version out to Nuget.
